src/components/megalinks/components/megalink-multi-image/MegalinkMultiImage.vue

```diff
@@ -222,33 +222,18 @@ export default {
         }
 
         .vs-stretched-link-card__video-button {
-            position: absolute;
-            bottom: 100%;
+            float: left;
```

Comment:
Not sure this float is doing anything as the video-button is still picking up the position absolute from the vs-stretched-link-card